devenv/main.py

```diff
@@ -19,6 +20,10 @@
 
 
 def devenv(argv: Sequence[str], config_path: str) -> ExitCode:
+    if len(argv) > 1 and argv[1] in ("version", "--version"):
+        print(f"devenv {version}")
+        return 0
+
```

Comment:
As is, this won't show up in help since we aren't making use of argparse. We should make this work more cohesive similar to the rest of the commands, even if it is simpler.

Reply:
The problem is, if we have `parser.add_argument("--version", action="store_true")`

then `devenv --version` gives:

```
$ devenv --version
usage: devenv [-h] [--version] command ...
devenv: error: the following arguments are required: command
```

I just gave up quickly and opted for this instead. I think it's fine.

Reply:
I think it would be worthwhile to spend the time to figure out how to do this using argparse so we can provide a cohesive cli where every command is listed in the help. If this PR needs to get out quick, I'd suggest leaving this part off and addressing it in a separate PR.

Reply:
Yeah, i'd like to get this out quick so I can unblock a new release + remaining sentry cleanup. Can polish this stuff later.